pdr_backend/ppss/test/test_predictoor_ss.py

```diff
@@ -63,6 +63,21 @@ def test_predictoor_ss_test_dict():
     assert d["aimodel_ss"]["input_feeds"] == ["binance ETH/USDT c 1h"]
 
 
```

Comment:
You don't need a dedicated test for this. It adds more complexity than needed.

Rather, incorporate this into the existing tests surgically.